packages/flutter/lib/src/widgets/navigation_toolbar.dart

```diff
@@ -106,7 +106,7 @@ class _ToolbarLayout extends MultiChildLayoutDelegate {
 
     if (hasChild(_ToolbarSlot.leading)) {
       final BoxConstraints constraints = BoxConstraints(
```

Comment:
More of a question, should this just be `BoxConstraints.loose(size)` to match the trailing widget? @xster

Reply:
I think it likely can be (if all the tests still pass)